### ISSUE TITLE
<table> not being generated

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#585]
The page `en.wikisource.org/wiki/Page:A_simplified_grammar_of_the_Swedish_language.djvu/20` is not generating a table (as intended)
I suspect this is a subtle issue with \<noinclude>

The first two lines of wikitext are:

    <noinclude><pagequality level="4" user="Jafeluv" />{{rh|left=2|center=SWEDISH GRAMMAR.}}</noinclude>:{|
    |- valign="top"

If I were to amend this to

    :{|
    |- valign="top"
That is, remove the \<noinclude> section manually
Then a table is produced

**Comment by gnosygnu** [comment]
Yeah, the problem here is the XOWA template parser. It tries to squeeze everything into one-pass whereas the MediaWiki parser does multi-passes.

In the above case, the &lt;noinclude> is processed first which leaves ```START_OF_DOCUMENT:{|```. XOWA just sees ```</noinclude>:{|``` and disregards the wikitext syntax for table (tables need to be preceded by either a new line or the start of the document)

Let me know if you see this on many pages, and I can try to put in a limited fix for this. Otherwise this falls into the bucket of template parser rewrite

Thanks.
